core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -81,7 +81,7 @@ private static String prependTitleIfRequired(String icon, String title) {
 
     // for Jelly
     @Restricted(NoExternalUse.class)
-    public static String getSymbol(String name, String title, String tooltip, String classes, String pluginName) {
+    public static String getSymbol(String name, String title, String tooltip, String classes, String pluginName, String id, String style) {
```

Comment:
do you need style? inline style isn't recommended (that's the main thing holding me back from approving this).

Also what do you want the ID for?

Reply:
For #6498 I want to use an a clickable chevron up/down to display additional labels. To be able to change visibility of the icon I need it's id.

The style was just for completeness as the icon taglib explicitly allows to set a style, but I do not need it for my purposes

Reply:
could you remove the style please?

do you need id or could you use class? most of Jenkins works by using class selector's so that JavaScript is applied to all matching elements, but we can add ID if you need it

Reply:
in my use case I need to identify the icon for a certain node on the /computer page. To use the class I would need to create a class containing the computer name, but that can probably lead to problems. Using an ID is I think the cleaner way